chrome/tweaks/collapse-sidebery.css

```diff
@@ -0,0 +1,77 @@
+@media (-moz-bool-pref: "uc.tweak.collapse-sidebery") {
+  :root:not([customizing]):has(#sidebar-box[sidebarcommand="_3c078156-979c-498b-8990-85f7987dd929_-sidebar-action"]:not([hidden])) {
+    --uc-sidebar-width: 256px;
+    --uc-sidebar-collapsed-width: 48px;
+
+    & #sidebar-box {
+      position: relative;
+      min-width: var(--uc-sidebar-collapsed-width) !important;
+      width: var(--uc-sidebar-collapsed-width) !important;
+      max-width: var(--uc-sidebar-width) !important;
+      z-index: calc(var(--browser-area-z-index-tabbox, 0) + 1) !important;
+    }
+
+    & #sidebar-box[positionend] {
+      direction: rtl;
+    }
+    & #sidebar-box[positionend] > * {
+      direction: ltr;
+    }
+
+    & #sidebar-box[positionend]:-moz-locale-dir(rtl) {
+      direction: ltr;
+    }
+    & #sidebar-box[positionend]:-moz-locale-dir(rtl) > * {
+      direction: rtl;
+    }
+
+    & #sidebar-box[positionend] {
+      & #sidebar {
+        border-left: 1px solid var(--sidebar-border-color);
+      }
+    }
+    & #sidebar-box:not([positionend]) {
+      & #sidebar {
+        border-right: 1px solid var(--sidebar-border-color);
+      }
+    }
+
+    & #sidebar-header {
+      display: none;
+    }
+
+    & #sidebar {
+      background-color: inherit;
+      overflow: hidden;
+      padding-inline: 4px;
+      width: var(--uc-sidebar-width) !important;
+    }
+
+    & #sidebar-splitter {
+      display: none;
+    }
+  }
+
+  :root:not([customizing]):has(#sidebar-title[value="Sidebery"]:not([hidden])),
```

Comment:
I think it would be best to change this to use the same selector as above, `#sidebar-box[sidebarcommand="_3c078156-979c-498b-8990-85f7987dd929_-sidebar-action"]` as the panel name can be arbitrary based on user settings.

Reply:
Updated